bundles/FileExplorerBundle/public/js/startup.js

```diff
@@ -44,7 +44,7 @@ pimcore.bundle.file_explorer.startup = Class.create({
         var perspectiveCfg = pimcore.globalmanager.get("perspective");
 
         if (
-            user.admin &&
+            (user.isAllowed('fileexplorer') || user.admin )&&
```

Comment:
The new permission was fine. Please add it back again, since we introduce the new permission in the installer.